iosxr_iso2vbox.py

```diff
@@ -229,44 +253,44 @@ def cleanup_vms(name, box_name):
     # Delete existing Box
     if os.path.exists(box_out):
         os.remove(box_out)
-        verboseprint('Found and deleted previous %s' % box_out)
+        logger.debug('Found and deleted previous %s' % box_out)
 
     # Delete existing OVA
     if os.path.exists(ova_out) and create_ova is True:
         os.remove(ova_out)
-        verboseprint('Found and deleted previous %s' % ova_out)
+        logger.debug('Found and deleted previous %s' % ova_out)
 
     # Destroy default vagrant box
-    verboseprint('Destroy default box')
-    run('vagrant destroy --force')
+    # logger.debug('Destroy default box')
+    # run('vagrant destroy --force')
 
     # Clean up existing vm's
     cleanup_vms(vmname, vbox)
 
     # Remove stale SSH entry
-    verboseprint('Removing stale SSH entries')
+    logger.debug('Removing stale SSH entries')
     run('ssh-keygen -R [localhost]:2222')
     run('ssh-keygen -R [localhost]:2223')
 
     # Create and register a new VirtualBox VM
-    verboseprint('Create VM')
+    logger.debug('Create VM')
     run('VBoxManage createvm --name %s --ostype Linux26_64 --basefolder %s' % (vmname, base_dir))
```

Comment:
just as a note - if you do as I suggested above and let your `run()` take a list of params, this would be IMHO clearer since it wouldn't need string substitution:

```python
run(['VBoxManage', 'createvm', '--name', vm_name,
     '--ostype', 'Linux26_64', '--basefolder', base_dir])
```